runner/ansible/roles/1.2.2/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Set Test ID
+  set_fact:
+    test_id: '1.2.2'
+
+- name: "{{ test_id }}.check"
+  shell: |
+   timeout=$(crm_attribute -t crm_config -G -n stonith-timeout --quiet)
+   if [[cibadmin -Q --xpath "//primitive[@type='fence_azure_arm']/@type" > /dev/null 2>&1 ]]; then
+     exit $([[ "${timeout}" = "{{ expected['1.2.2.fence_azure_arm'] }}" ]])
+   else
+     exit $([[ "${timeout}" = "{{ expected['1.2.2.sbd'] }}" ]])
+   fi
+  check_mode: no
+  register: config_updated
+  changed_when: config_updated.rc != 0
+  failed_when: config_updated.rc > 1
```

Comment:
I guess that  cosmetics depends on your point of view. Without failed_when: ansible will report a failed state and by default will terminate the run. We can ingore the "failure" which will let the play continue. But our convention currently is for failing tests to set changed status.

Reply:
Ok, so it is not cosmetics. Got it.
So what will happen when we have `config_updated.rc > 1` ?
Because this is a possible scenario, when the check fails